lib/web/scripts/node-join/install.sh

```diff
@@ -463,6 +463,9 @@ app_service:
   - name: "${APP_NAME}"
     uri: "${APP_URI}"
     public_addr: ${APP_PUBLIC_ADDR}
+    labels:{{range $index, $line := .appServerResourceLabels}}
```

Comment:
Do you need a conditional here to handle the case when labels is empty?

Reply:
we don't have to, this is valid `labels: ` (no values), i also tested it without adding labels to double check

Reply:
Some day (🤞) I will convert this script to go (the same thing we did for Server Auto Discovery scripts), and it will be much simpler to do these things